gtsam/linear/HessianFactor.cpp

```diff
@@ -467,7 +467,7 @@ std::shared_ptr<GaussianConditional> HessianFactor::eliminateCholesky(const Orde
     info_.choleskyPartial(nFrontals);
 
     // TODO(frank): pre-allocate GaussianConditional and write into it
-    const VerticalBlockMatrix Ab = info_.split(nFrontals);
+    VerticalBlockMatrix Ab = info_.split(nFrontals);
```

Comment:
Did this work? I didn't see a comment about it in the PR description.

Reply:
Too small to notice.